extension-ui/background.js

```diff
@@ -193,3 +193,198 @@ function showNotification(site, num, options = {}) {
   
   chrome.notifications.create(notificationOptions);
 }
+
+let timeoutId = null;
+let notificationTimeoutId = null;
+let remainingTime = initialTime;
+let lastVisitTime = Date.now();
+let lastVisitDate = new Date().toDateString();
```

Comment:
please join lastVisitTime & lastVisitDate to one object, lastVisit contains time & date properties